test/extended/oauth/redirect_uris.go

```diff
@@ -0,0 +1,144 @@
+package oauth
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+
+	"github.com/RangelReale/osincli"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
```

Comment:
nit: not a huge fan of single-character imports, do you have precendences elsewhere?

Reply:
This seems to be the convention around most e2e tests in this repo; keeping, as discussed.